go/oasis-test-runner/oasis/args.go

```diff
@@ -158,24 +159,47 @@ func (args *argBuilder) workerClientPort(port uint16) *argBuilder {
 	return args
 }
 
-func (args *argBuilder) workerRegistrySentryAddresses(addrs []string) *argBuilder {
+func (args *argBuilder) disableRegistrationWorker() *argBuilder {
+	args.vec = append(args.vec, []string{
+		"--" + registration.CfgRegistrationWorkerEnabled + "=false",
```

Comment:
Is this a new flag? It is not mentioned in the changelog fragment. Also, is this required because of #1930 (e.g., it could be handled automatically)?

Reply:
Oh forgot to add it to the changelog fragment. So it's not technically needed actually, i just wanted to disable the registration worker on sentry nodes as it's not needed (in case it's enabled the registrations are failing anyway, so no actual difference). So i'm fine with removing the flag and postponing this to the #1930  - i don't see a reason it couldn't be done automatically (maybe by checking node roles?).

Reply:
i removed it for now as it's not technically needed, and it can be properly fixed with #1930